staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -320,12 +322,12 @@ func NewCloud(configReader io.Reader) (cloudprovider.Interface, error) {
 				Duration: time.Duration(config.CloudProviderBackoffDuration) * time.Second,
 				Jitter:   config.CloudProviderBackoffJitter,
 			}
+			klog.V(2).Infof("Azure cloudprovider using backoff configuration: retries=%d, exponent=%f, duration=%d, jitter=%f",
```

Comment:
As far as I can tell we shouldn't log this if backoff is shouldOmitCloudProviderBackoff is true

Reply:
when shouldOmitCloudProviderBackoff==true, there's another backoff logic within Azure Go SDK. It's better to log the backoff parameters also for it.